shipyard-controller/debugdocs/docs.md

```diff
@@ -0,0 +1,32 @@
+# Keptn Debug-UI
+
+The Debug-UI was built to debug keptn.
+
+**Features:**
+ - See all sequences of a project 
+ - Get all the events of a specific sequence 
+ - Get the sequences which are blocking a specific sequence 
+ - Export a MongoDB collection
+
+The Debug-UI can be exposed on port 9090 of the shipyard controller by setting 'DEBUGUI_ENABLED' in the deploy/service.yml to "true".
```

Comment:
instead of referring specifically to the `deploy/service.yaml` I would suggest to rephrase it to something like "set the `DEBUGUI_ENABLED environment variable to `"true"`"